app/src/main/java/com/pysiak/simple/clock/ui/SimpleClockActivity.java

```diff
@@ -0,0 +1,56 @@
+package com.pysiak.simple.clock.ui;
+
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.design.widget.TabLayout;
+import android.support.v4.app.Fragment;
+import android.support.v4.view.ViewPager;
+import android.support.v7.app.AppCompatActivity;
+import android.widget.Toast;
+import com.pysiak.simple.clock.R;
+
+import java.util.List;
+
+/**
+ * Main class, launcher activity for application, implementation of SimpleClockView
+ * @author Yaroslav P.
+ * {@link AppCompatActivity}
+ * {@link SimpleClockView}
```

Comment:
Move this link to description part - replace "...implementation of SimpleClockView" to "...implementation of {@link SimpleClockView}"